CHANGES.rst

```diff
@@ -22,7 +22,8 @@ Bug Fixes
 
 - ``ccd_process`` now copies the meta of the input when subtracting the
   master bias. [#404]
-
```

Comment:
It can include both: `[#421, #422]`. If you want to do just one, I'd say do the PR.